xjc-maven-plugin/src/main/java/com/github/davidmoten/xjc/XjcMojo.java

```diff
@@ -296,7 +296,7 @@ private File createOutputDirectoryIfSpecifiedOrDefault() {
             }
         }
 
-        final File outputDir = new File(mavenProject.getBuild().getDirectory());
+        final File outputDir = new File(".");
```

Comment:
nit: just return `new File(".")`. We don't need to ensure that the project directory exists.

Reply:
Nice catch, will fix it